docs/reference/controls/colorpicker/README.md

```diff
@@ -0,0 +1,113 @@
+
+Main improvements learning from WinUI were:
+ * The `ColorPicker` is implemented as a drop-down (matching all other "pickers"). There is also a `ColorView` control for those that want the canvas-type control (similar to WinUI).
+ * The Avalonia controls attempt to do everything possible in XAML control themes keeping code-behind to an absolute minimum. This significantly increases composability and enables app developers to customize every part of these controls (and even the primitives in most cases).
+ * Primitives such as the `ColorSlider` and `ColorSpectrum` are fully self-contained and can be used separately enabling app developers to create custom color picker implementations.
+ * A new `HsvColor` struct was added to base Avalonia itself (alongside `Color` and `HslColor`) and is now used in all color picker controls. This simplified code-behind and also made binding of color properties between primitives and controls possible. Color picker controls internally work in HSV color space.
+ * `HsvColor` along with `ColorSlider` together unlock a lot of power compared to WinUI (and enable easy re-templating).
+ * Many new properties (more than in WinUI) were added to control all aspects of the `ColorView` visibility. Each tab can be separately hidden along with most individual subsections. This allows a lot of design customization without having to re-template or use complex style selectors.
+ * Color palettes were added using the `IColorPalette` interface (same as the Windows Community Toolkit). No color palettes are supported in the WinUI version of this control.
+ * New properties such as `SelectedIndex` and `ColorModel` allow customizing the color picker and putting it into a pre-defined state. For example, the WinUI ColorPicker always defaults to RGB and this cannot be changed in code or XAML. This implementation does not have such limitations.
+
+## Controls & Primitives
+
+| Control          | Link                                           |
+|------------------|------------------------------------------------|
+| `ColorPicker`    |        |
+| `ColorView`      | See the dedicated [`ColorView`](colorview.md) page. |
+| `ColorSpectrum`  |        |
+| `ColorSlider`    |        |
+| `ColorPreviewer` |        |
+
+## Color Palettes
+
+Several pre-defined color palettes implementing the `IColorPalette` interface are provided. Instances of these palettes may be set to the `Palette` property of a `ColorView` or `ColorPicker`.
+
+<table>
+  <tr>
+    <th>Palette</th>
+    <th>Description</th>
+  </tr>
+  <tr>
+    <td>
+      <img src="/img/controls/colorpicker/color-palette-fluent.png" alt="Fluent Color Palette" width="300" />
```

Comment:
For images and links, we now use JS import syntax, so build tool can validate path in compile time. See https://github.com/AvaloniaUI/avalonia-docs/blob/main/docs/reference/controls/buttons/button.md?plain=1#L5

Also, we were slowly moving from HTML to Markdown on some other pages. But it's not a bit deal.

Reply:
Images in markdown tables may not always be supported. I think I ran into an issue in the past which required this. I'll use the standard markdown format though and we'll see if it works here.

Reply:
Made both changes

Reply:
@maxkatz6 In the published docs only the first color palette image is shown. This type of problem is why the table was HTML.

Is there a way to fix this or should I revert back to HTML here?

Reply:
I'm reverting back to HTML here: https://github.com/AvaloniaUI/avalonia-docs/pull/238